plugins/modules/iosxr_config.py

```diff
@@ -451,6 +457,11 @@ def main():
 
     if module.params["force"] is True:
         module.params["match"] = "none"
+    if (
```

Comment:
So if disable_default_comment is True and a comment is explicitly set, the comment will be applied?

Reply:
yes it will be applied.